### ISSUE TITLE
Address Book Autocompletion

### ISSUE THREAD

**Issue by Lantizia** [Ericsson/exchangecalendar#113]
Hi,

I'm thrilled to report I've got access to my Office 365 contacts by using the "Add Exchange contact folder" featrure you see inside Thunderbird's Address Book -- additionally even the 'Add global address list to search results' option works too!

I'm curious what is preventing the ability to do name/e-mail autocompletion from the To: / CC: line from the address books you can add.  It seems only 'Local Address Books' and 'Directory Server' can be used in Thunderbird for Autocompletion - but 'Directory Server' seems only to be for LDAP.

So basically it looks like the bulk of the work is done, just the ability to search/autocomplete from writing a new e-mail is missing.

Any thoughts?

UPDATE: It appears to be magically working now, didn't notice but it seems the autocomplete shows results from this add-on with a little yellow Outlook icon.  I'm guessing this is activated automatically as soon as the add-on is enabled, no option to turn it off or on?

**Comment by muthusuba** [comment]
@Lantizia Its nice to hear that its working for you. Maybe you could help write a wiki page for us, please? I'm sure it'll help many...Thanks :)
